api/api.go

```diff
@@ -231,31 +232,52 @@ type ProvisionersResponse struct {
 	NextCursor   string
 }
 
+const redacted = "*** REDACTED ***"
+
+func scepFromProvisioner(p *provisioner.SCEP) *models.SCEP {
+	return &models.SCEP{
+		ID:                            p.ID,
+		Type:                          p.Type,
+		Name:                          p.Name,
+		ForceCN:                       p.ForceCN,
+		ChallengePassword:             redacted,
+		Capabilities:                  p.Capabilities,
+		IncludeRoot:                   p.IncludeRoot,
+		ExcludeIntermediate:           p.ExcludeIntermediate,
+		MinimumPublicKeyLength:        p.MinimumPublicKeyLength,
+		DecrypterCertificate:          []byte(redacted),
+		DecrypterKeyPEM:               []byte(redacted),
+		DecrypterKeyURI:               redacted,
+		DecrypterKeyPassword:          []byte(redacted),
```

Comment:
I've fixed this in https://github.com/smallstep/certificates/pull/1554

Reply:
The idea was to use a separate model for the public API response, so that `*** REDACTED ***` would show up instead of the base64 encoding of that. So we may want to revisit this, and make the CLI use the "public" model instead. But there's probably more to be discussed around that, since I only did this for this single provisioner/endpoint.